src/cascadia/TerminalSettingsEditor/Profiles.idl

```diff
@@ -74,6 +79,9 @@ namespace Microsoft.Terminal.Settings.Editor
         ProfileViewModel Profile;
         ProfilesPivots LastActivePivot;
 
+        void CreateUnfocusedAppearance();
+        void DeleteUnfocusedAppearance();
+
```

Comment:
You can probably remove these two functions and just go through the profile instead, right? These don't feel like they should be a part of the navigation state.

Reply:
Unfortunately we need some way to pass the list of color schemes and the window root to the profile view model for it to create an appearance view model. The PVM doesn't have the list of color schemes nor the window root, only the navigation state does.